server/utils/licenceStatus.js

```diff
@@ -236,10 +224,7 @@ function getEligibilityStageState(licence) {
       addressUnsuitable,
     },
     tasks: {
-      exclusion,
-      crdTime,
-      suitability,
-      eligibility,
+      ...eligibilityState.tasks,
```

Comment:
I could very easily be talked out of this destructuring and instead being more explicit about whats being passed through